Build failure: libe-book (cross-compilation)

https://github.com/NixOS/nixpkgs/blob/nixpkgs-unstable/pkgs/by-name/li/libe-book/package.nix

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

```
nix-build '<nixpkgs>' -I nixpkgs=channel:nixos-unstable --arg crossSystem '{ config = "aarch64-unknown-linux-gnu"; }' -A libe-book
```

### Relevant log output

```console
checking for gperf... no
configure: error: gperf not found
```

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.15.1, NixOS, 25.05 (Warbler), 25.05.803579.70c74b02eac4`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.3`
 - channels(root): `"nixos-25.05, nixos-hardware"`
 - nixpkgs: `/nix/store/gdz4yp5sjq3b1q0g9ad9hicqsx1alvif-nixos-25.05/nixos`

### Notify maintainers

@7c6f434c

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/li/libe-book/package.nix
+++ b/pkgs/by-name/li/libe-book/package.nix
@@ -28,10 +28,13 @@ stdenv.mkDerivation rec {
       "TRUE, TRUE, &status)" \
       "true, true, &status)"
   '';
-  nativeBuildInputs = [ pkg-config ];
+
+  nativeBuildInputs = [
+    pkg-config
+    gperf
+  ];
 
   buildInputs = [
-    gperf
     librevenge
     libxml2
     boost

